CHANGELOG.md

```diff
@@ -8,15 +8,35 @@
+- ([\#1043](https://github.com/mesg-foundation/engine/pull/1043)) New gRPC Execution APIs and SDK. ([\#1052](https://github.com/mesg-foundation/engine/pull/1052)). ([\#1064](https://github.com/mesg-foundation/engine/pull/1064)). ([\#1065](https://github.com/mesg-foundation/engine/pull/1065)). ([\#1070](https://github.com/mesg-foundation/engine/pull/1070)). ([\#1124](https://github.com/mesg-foundation/engine/pull/1124)). ([\#1132](https://github.com/mesg-foundation/engine/pull/1132)). ([\#1135](https://github.com/mesg-foundation/engine/pull/1135)).
+- ([\#1053](https://github.com/mesg-foundation/engine/pull/1053)) New gRPC Event APIs and SDK. ([\#1054](https://github.com/mesg-foundation/engine/pull/1054)). ([\#1073](https://github.com/mesg-foundation/engine/pull/1073)). ([\#1126](https://github.com/mesg-foundation/engine/pull/1126)). ([\#1144](https://github.com/mesg-foundation/engine/pull/1144)). ([\#1141](https://github.com/mesg-foundation/engine/pull/1141)).
+
+#### Changed
+
+- ([\#1082](https://github.com/mesg-foundation/engine/pull/1082)) Server and SDK package cleanup. ([\#1085](https://github.com/mesg-foundation/engine/pull/1085)). ([\#1096](https://github.com/mesg-foundation/engine/pull/1096)).
+- ([\#1151](https://github.com/mesg-foundation/engine/pull/1151)) Namespace simplification in package Container.
+- ([\#1094](https://github.com/mesg-foundation/engine/pull/1094)) Introduction of a Hash type to manage all encoding and decoding of hashes. ([\#1072](https://github.com/mesg-foundation/engine/pull/1072)). ([\#1098](https://github.com/mesg-foundation/engine/pull/1098)).
+- ([\#1148](https://github.com/mesg-foundation/engine/pull/1148)) Update EthWallet to use new gRPC APIs.
```

Comment:
what about the system service marketplace and the new way to have the system service

Reply:
Have docs been updated about this? The changelog is one place, but what about docs/tutoria?

Reply:
no we need to update the doc as well but we can still release without the update and we update the doc when we update the CLI too